stages/apps/sharedsvc/outputs.tf

```
@@ -18,6 +18,14 @@ output "keycloak_hostname" {
   value = module.keycloak.keycloak_hostname
 }
 
+output "harbor_hostname" {
+  value = module.harbor.hostname
```

`module.harbor` has the `count` argument now, so I think we'll have to do this instead:

```suggestion
  value = module.harbor[0].hostname
```